websites/A/ANIMETV7/presence.ts

```diff
@@ -0,0 +1,188 @@
+const presence = new Presence({
+		clientId: "1106959657735561256",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/CtyAaoq.jpg",
```

Comment:
Same here.
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

Reply:
check now